src/tests/unittests/validation/BufferValidationTests.cpp

```diff
@@ -270,23 +383,105 @@ TEST_F(BufferValidationTest, DISABLED_MapReadDestroyBeforeResult) {
 }
 
 // When a request is cancelled with Unmap it might still be in flight, test doing a new request
-// works as expected and we don't get the cancelled request's data.
+// works as expected and we don't get the cancelled request's data. Reading part
```

Comment:
Incomplete sentence? Or do you mean, like, `(Reading part.)`? In which case I would say:
```
When a MapRead is cancelled (...)
```

Reply:
Done